astropy/visualization/mpl_normalize.py

```diff
@@ -130,7 +130,7 @@ def inverse(self, values):
         return values_norm * (self.vmax - self.vmin) + self.vmin
 
 
-def simple_norm(data, stretch='linear', power=1.0, asinh_a=0.1, min_cut=None,
+def simple_norm(data, stretch='linear', power=1.0, asinh_a=0.1, log_a=1000, min_cut=None,
```

Comment:
Please add the new argument as the last one to keep backward compatibility. Also keep the default on the value as it was previously.

Reply:
Definitely! Although, what do you mean keep the default on the value as it was previously? I am asking because there wasn't a default on the log value.

Reply:
`LogStretch()` must have assumed a value for it? Maybe it didn't.

Reply:
`class LogStretch(BaseStretch):
    r"""
    A log stretch.

    The stretch is given by:

    .. math::
        y = \frac{\log{(a x + 1)}}{\log{(a + 1)}}.

    Parameters
    ----------
    a : float
        The ``a`` parameter used in the above formula.  Default is 1000.
    """

    def __init__(self, a=1000.0):
        super().__init__()
        self.exp = a

    def __call__(self, values, clip=True, out=None):
        values = _prepare(values, clip=clip, out=out)
        np.multiply(values, self.exp, out=values)
        np.add(values, 1., out=values)
        np.log(values, out=values)
        np.true_divide(values, np.log(self.exp + 1.), out=values)
        return values

    @property
    def inverse(self):
        """A stretch object that performs the inverse operation."""
        return InvertedLogStretch(self.exp)`

Reply:
OK, this is good then. 👍